inc/customizer/react/src/divider/divider-component.js

```diff
@@ -0,0 +1,52 @@
+import PropTypes from 'prop-types';
+
+const { __ } = wp.i18n;
+const { Component, Fragment } = wp.element;
```

Comment:
@abhishek-bsf Please replace above two lines with 

```suggestion
import { Component, Fragment } from '@wordpress/element';
```

Reply:
Resloved!